starboard/shared/modular/cobalt_layer_posix_pipe2_abi_wrappers.cc

```diff
@@ -0,0 +1,22 @@
+// Copyright 2025 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+extern "C" {
+
+int __abi_wrap_pipe2(int fildes[2], int flag);
```

Comment:
Do we often use `fildes` as a shortening of file descriptors? I think my preference would be to use the pipe2 argument parameter names: `pipefd` and `flags`.

Reply:
Yep that sounds good!